config/schemas/form_response.json

```diff
@@ -0,0 +1,329 @@
+          "Care for children"
+        ]
+      }
+    },
+    "offer_care_qualifications": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "DBS check",
+          "Nursing or other healthcare qualification",
+          "I do not have a qualification"
+        ]
+      }
+    },
+    "offer_care_qualifications_type": {
+      "type": ["string", "null"]
+    },
+    "offer_other_support": {
+      "type": "string"
+    },
+    "location": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "East of England",
+          "East Midlands",
+          "West Midlands",
+          "London",
+          "North East",
+          "North West",
+          "South East",
+          "South West",
+          "Yorkshire and the Humber",
+          "Northern Ireland",
+          "Scotland",
+          "Wales"
+        ]
+      }
+    },
+    "business_details": {
+      "type": "object",
+      "required": [
+        "company_name",
+        "company_size",
+        "company_location"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "company_name": {
+          "type": "string"
+        },
+        "company_number": {
+          "type": ["string", "null"]
+        },
+        "company_size": {
+          "type": "string",
+          "enum": [
+            "Under 50 people",
+            "50 to 250 people",
+            "More than 250 people"
+          ]
+        },
+        "company_location": {
+          "type": "string",
+          "enum": [
+            "United Kingdom",
+            "European Union",
+            "Rest of world"
+          ]
+        },
+        "company_postcode": {
+          "$ref": "#/definitions/postcode"
+        }
+      }
+    },
+    "contact_details": {
+      "type": "object",
+      "required": [
+        "contact_name",
+        "phone_number",
+        "email"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "contact_name": {
+          "type": "string"
+        },
+        "role": {
+          "type": ["string", "null"]
+        },
+        "phone_number": {
+          "type": "string"
+        },
+        "email": {
+          "type": "string",
+          "pattern": "[@]{1}"
+        }
+      }
+    },
+    "reference_number": {
+      "type": "string"
+    },
+    "check_answers_seen": {
+      "type": "boolean"
+    },
+    "_csrf_token": {
+      "type": "string"
+    },
+    "current_path": {
+      "type": "string"
+    },
+    "previous_path": {
+      "type": "string"
+    },
+    "additional_product": {
+      "$ref": "#/definitions/yes_no"
+    }
+  },
+  "definitions": {
+    "yes_no": {
+      "type": "string",
+      "enum": ["Yes", "No"]
+    },
+    "postcode": {
+      "type": ["string", "null"],
+      "pattern": "^[A-Za-z0-9 ]{4,}$"
+    },
+    "whole_number": {
+      "type": "string",
+      "pattern": "^[0-9]+$"
+    }
+  }
+}
```

Comment:
This looks great 🎉 

Since we might change the translations file without updating this one, that might cause some interesting issues.

One thing that might be interesting is to create a `form_response.json.erb` that we can use to generate a valid schema from `en.yml` changes. That might prevent us from shipping a change that allows users to submit invalid data and highlight in pull requests that the schema is changing, so we can version it / let stakeholders know. What do you think?

Reply:
We discussed offline and decided we'll do this in another PR (might need to do a bit of cleanup of the en.yml file).